CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` UDF to distribute tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` UDF that allows
+  splitting a shard to specified set of nodes without blocking writes
+  and based on given split points
+
+* Adds support for non-blocking tenant isolation
+
+* Adds support for isolation tenants that use partitioned tables
+  or columnar tables
+
+* Drops support for isolation tenants that use replicated tables
+
+* Separates columnar table access method into a separate logical extension
+
+* Adds support for logical replication in `replicate_reference_tables()`
+
+* Improves performance of blocking shard moves
+
+* Improves non-blocking shard moves with a faster custom copy logic
+
+* Adds the GUC `allow_unsafe_constraints` to allow constraints without
+  distribution column
+
+* Introduces GUC `citus.skip_constraint_validation`
+
+* Replaces `citus.hide_shards_from_app_name_prefixes` GUC with
```

Comment:
Isn't this 11.0 item? https://docs.citusdata.com/en/v11.0/develop/api_guc.html?highlight=show_shards_for_app_name_prefixes#citus-show-shards-for-app-name-prefixes-text


 And, that was an internal change within the release, so I don't think we should have this here